README.md

```diff
@@ -1,24 +1,67 @@
 # shavar-prod-lists
-This repo serves as a staging area for shavar/tracking protection lists prior to production deployment to Firefox. This repo gives Mozilla a chance to manually review all updates before they go live, a fail-safe to prevent accidental deployment of a list that could break Firefox.
+This repo serves as a staging area for [shavar](https://github.com/mozilla-services/shavar)/[tracking protection](https://wiki.mozilla.org/Security/Tracking_protection) lists prior to [production deployment to Firefox](https://mana.mozilla.org/wiki/display/SVCOPS/Shavar+-+aka+Mozilla's+Tracking+Protection). This repo gives Mozilla a chance to manually review all updates before they go live, a fail-safe to prevent accidental deployment of a list that could break Firefox.
 
 
-### Raw lists
+## Lists
 These two JSON files power Tracking Protection in Firefox. 
 
-* disconnect-blacklist.json is a version controlled copy of the Disconnect.me public GPL anti-tracking list available here: https://services.disconnect.me/disconnect-plaintext.json
-* disconnect-entitylist.json is a transformed version of the public GPL list that Disconnect reformats for Firefox, in order to reduce the size of the entity list we send to Firefox via the Shavar service. 
+* `disconnect-blacklist.json` is a version controlled copy of the Disconnect.me public GPL anti-tracking list available here: [https://services.disconnect.me/disconnect-plaintext.json](https://services.disconnect.me/disconnect-plaintext.json)
```

Comment:
You can also use `<http://example.com>` in markdown linkify URLs without having to repeat them.